database/src/interface/UserInterface.ts

```diff
@@ -6,8 +6,8 @@ export interface UserInterface {
   username?: string
   description?: string
   password?: BigInt
-  pubKey?: Buffer
-  privKey?: Buffer
+  pubKey?: Buffer | null
+  privKey?: Buffer | null
```

Comment:
The questionmak allows to leave the value - the definition of null should not be required

Reply:
Same above

Reply:
`should` is not `is`
I had to change this to get the typescript compiled

Reply:
```
TSError: ⨯ Unable to compile TypeScript:
src/seeds/users/garrick-ollivander.seed.ts:7:31 - error TS2345: Argument of type '{ email: string; firstName: string; lastName: string; username: string; description: string; password: bigint; pubKey: null; privKey: null; emailHash: Buffer; createdAt: Date; emailChecked: boolean; ... 8 more ...; addBalance: boolean; }' is not assignable to parameter of type 'UserInterface'.
  Types of property 'pubKey' are incompatible.
    Type 'null' is not assignable to type 'Buffer | undefined'.
```